src/Confluent.Kafka/AdminClient.cs

```diff
@@ -830,7 +887,38 @@ private Task StartPollTask(CancellationToken ct)
                                         }
                                         break;
                                     }
+                                    case Librdkafka.EventType.DescribeUserScramCredentials_Result:
+                                    {
+                                        if (errorCode != ErrorCode.NoError)
+                                        {
+                                            Task.Run(() =>
+                                                    ((TaskCompletionSource<DescribeUserScramCredentialsResult>)adminClientResult).TrySetException(
+                                                        new KafkaException(kafkaHandle.CreatePossiblyFatalError(errorCode, errorStr))));
+                                                break;
+                                        }
+                                        var result = extractDescribeUserScramCredentialsResult(eventPtr);
+                                        Task.Run(() =>
+                                                    ((TaskCompletionSource<DescribeUserScramCredentialsResult>)adminClientResult).TrySetResult(
+                                                        new DescribeUserScramCredentialsResult(){Error = result.Error, UserScramCredentialsDescriptions = result.UserScramCredentialsDescriptions}));
+                                        break;
```

Comment:
You aren't throwing `DescribeUserCredentialsException` anywhere?
```suggestion
case Librdkafka.EventType.DescribeUserScramCredentials_Result:
{
    if (errorCode != ErrorCode.NoError)
    {
        Task.Run(() =>
                ((TaskCompletionSource<DescribeUserScramCredentialsResult>)adminClientResult).TrySetException(
                    new KafkaException(kafkaHandle.CreatePossiblyFatalError(errorCode, errorStr))));
            break;
    }
    var results = extractDescribeUserScramCredentialsResult(eventPtr);
    if (results.Error != null && results.Error.IsError)
    {
        Task.Run(() =>
                ((TaskCompletionSource<DescribeUserScramCredentialsResult>)adminClientResult).TrySetException(
                    new DescribeUserScramCredentialsException(results)));
    }
    else
    {
        Task.Run(() =>
            ((TaskCompletionSource<DescribeUserScramCredentialsResult>)adminClientResult).TrySetResult(
                new DescribeUserScramCredentialsResult() { UserScramCredentialsDescriptions = results.UserScramCredentialsDescriptions }
        ));
    }
    break;
}
```